src/components/shared/buttons/MoveButton/MoveButton.tsx

```diff
@@ -0,0 +1,25 @@
+import Arrow from '../../../../icons/Arrow';
+import ArrowDark from '../../../../icons/ArrowDark';
+import IconButton from '../IconButton';
+
+interface Props {
+  isDisabled?: boolean;
+  iconClassName?: string;
+  onClick: () => void;
+}
+
+const MoveButton: React.FC<Props> = ({
+  isDisabled,
+  iconClassName,
+  onClick,
+}) => {
+  const ArrowIcon = isDisabled ? Arrow : ArrowDark;
+
+  return (
+    <IconButton isDisabled={isDisabled} onClick={onClick}>
+      <ArrowIcon className={iconClassName} />
+    </IconButton>
```

Comment:
If we see the design sigma, something going wrong, when icon is disabled: 
<img width="279" alt="image" src="https://github.com/fe-feb23-code-hunters/fe_phone-catalog/assets/113451960/0d62dba3-fdc7-4bdd-a640-04eb07032d1a">

And Figma design: 
<img width="895" alt="image" src="https://github.com/fe-feb23-code-hunters/fe_phone-catalog/assets/113451960/436e543c-3eb4-4f04-88a6-7cb6511ef308">

Reply:
@Hamuud Check one more time please